code/game/objects/items/holy_weapons.dm

```diff
@@ -390,8 +391,31 @@
 	lefthand_file = 'icons/mob/inhands/weapons/polearms_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/polearms_righthand.dmi'
 	name = "unholy pitchfork"
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
 	desc = "Holding this makes you look absolutely devilish."
 	attack_verb = list("poked", "impaled", "pierced", "jabbed")
+	slot_flags = SLOT_BACK
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	sharpness = IS_SHARP
+	force = 10
+	bonus_burn = 5
+	
+/obj/item/nullrod/pitchfork/clockworkspear/clockwork 
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "ratvarian_spear"
+	item_state = "ratvarian_spear"
+	lefthand_file = 'icons/mob/inhands/antag/clockwork_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/antag/clockwork_righthand.dmi'
+	name = "clockwork spear"
+	desc = "A pointy spear made of holy brass. It ticks and tocks."
+	attack_verb = list("ticked", "tocked", "singed", "speared")
+	var/clockwork_desc = "A disgusting and cheap replica created by the church to imitate the divine armory of the Justiciar."
```

Comment:
This variable does absolutely nothing. Either remove it or use it. You could do something like this:

```
/obj/item/nullrod/pitchfork/clockworkspear/clockwork(mob/user)
. = ..()
if(*some check to see if user is a clockcultist*)
    to_chat(user, clockwork_desc)
```

Not sure about the exacts there, but look around for a different `examine()` proc implementation and you should see what you should do here.

Reply:
The spear should be on the list of clockwork items, which means that Ratvarian cultists would get an alternate examine.